runner/ChallengeSession.feature

```diff
@@ -0,0 +1,102 @@
+Feature: Should allow the user to interact with the challenge server
+
+  Background:
+    Given There is a challenge server running on "localhost" port 8222
+    And journeyId is "aJourneyId"
+    And the challenge server exposes the following endpoints
+      | verb       | endpointEquals               | status   | responseBody                         | acceptHeader  |
+      | GET        | /availableActions/aJourneyId | 200      | Available actions coming from server | text/coloured |
+      | GET        | /roundDescription/aJourneyId | 200      | RoundID\nRound Description           | text/coloured |
+      | GET        | /journeyProgress/aJourneyId  | 200      | Journey progress coming from server  | text/coloured |
+
+    And the challenge server exposes the following endpoints
```

Comment:
Why do we need this as a separate And?

Reply:
One matches the url based on a regular expression, the other matches exactly. They require different methods in wiremock.
We can probably do a check to see if we detect if the url contains a regular expression, but that felt a bit error prone.

Reply:
Alternatively if we can match without a regular expression, then we can also get rid of this.